client/my-sites/people/invite-people/index.jsx

```diff
@@ -361,7 +359,8 @@ const InvitePeople = React.createClass( {
 									disabled={ this.state.sendingInvites } />
 								<FormSettingExplanation>
 									{ this.translate(
-										'(Optional) You can enter a custom message of up to 500 characters that will be included in the invitation to the user(s).'
+										'(Optional) You can enter a custom message of up to 500 characters ' +
```

Comment:
Yeah, please don't use template literals for this. The way it is split into multiple lines right now is the way to go. See https://github.com/Automattic/eslint-plugin-wpcalypso/blob/master/docs/rules/i18n-no-newlines.md

Thanks for the cc!